packages/providers/fal/avatar-generation/fal_avatar/models/base.py

```
@@ -0,0 +1,223 @@
+            duration: Duration in seconds
+
+        Returns:
+            Validated duration
+
+        Raises:
+            ValueError: If duration exceeds max
+        """
+        if duration > self.max_duration:
+            raise ValueError(
+                f"Duration {duration}s exceeds maximum {self.max_duration}s"
+            )
```

Ruff (0.14.10)

Avoid specifying long messages outside the exception class

(TRY003)